### PR TITLE
style(web, novui): Add custom scrollbar styling and remove workflow preview overflow

### PR DESCRIPTION
**Pull request by rifont**
### What changed? Why was the change needed?

* Remove overflow from `PageContainer` to remove scrolling by default
  * Add `overflowY: auto` to pages requiring scrollable content
* Add Panda Global CSS for scrollbar styling

### Screenshots
_Dark Mode_
image

_Light Mode_
image

_Recording demonstrating correct scrolling across the application_
https://www.loom.com/share/01e3c4e328cc42c489f7ca31343c70eb?sid=416b7fce-083c-4d61-a284-eefbb016cefa

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

File: apps/web/src/studio/layout/PageContainer.tsx [hunk 1]
```diff
@@ -10,7 +10,6 @@ export const PageContainer: FC<PropsWithChildren<IPageContainerProps>> = ({ chil
     <Container
       className={cx(
         css({
-          overflowY: 'auto !important',
```
Comment (rifont) [inline apps/web/src/studio/layout/PageContainer.tsx:13]: Removing overflow by default. Component consumers now need to explicitly specify when to overflow.

File: apps/web/src/pages/get-started/GetStartedPage.tsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ export function GetStartedPage() {
   };
 
   return (
-    <PageContainer>
+    <PageContainer className={css({ overflowY: 'auto' })}>
```
Comment (rifont) [inline apps/web/src/pages/get-started/GetStartedPage.tsx:34]: Here and below x2, explicitly setting overflow to be allowed.

File: libs/novui/src/global.styles.ts [hunk 3]
```diff
@@ -14,6 +15,11 @@ export const GLOBAL_CSS = defineGlobalStyles({
 
     backgroundColor: 'surface.page',
     overflow: 'hidden',
+    scrollbarWidth: 'thin',
+    scrollbarColor: {
+      base: `${LEGACY_COLOR_TOKENS.legacy.B80.value} transparent`,
+      _dark: `${LEGACY_COLOR_TOKENS.legacy.B30.value} transparent`,
```
Comment (rifont) [inline libs/novui/src/global.styles.ts]: Panda doesn't support token evaluation for CSS values requiring 2 or more properties to be specified inside a single string and doesn't have native support for these properties ([ref](https://panda-css.com/docs/utilities/interactivity)), so we need to use template literals instead.

File: libs/novui/src/global.styles.ts [hunk 4]
```diff
@@ -14,6 +16,11 @@ export const GLOBAL_CSS = defineGlobalStyles({
 
     backgroundColor: 'surface.page',
     overflow: 'hidden',
+    scrollbarWidth: 'thin',
+    scrollbarColor: {
+      base: `${token('colors.legacy.B80')} transparent`,
+      _dark: `${token('colors.legacy.B30')} transparent`,
```
Comment (rifont) [inline libs/novui/src/global.styles.ts]: Panda doesn't support token evaluation for CSS values requiring 2 or more properties to be specified inside a single string and doesn't have native support for these properties ([ref](https://panda-css.com/docs/utilities/interactivity)), so we need to use template literals instead.

File: apps/web/src/studio/components/workflows/step-editor/WorkflowStepEditorControlsPanel.tsx [hunk 5]
```diff
@@ -142,7 +142,7 @@ export const WorkflowStepEditorControlsPanel: FC<IWorkflowStepEditorControlsPane
 };
 
 export const formContainerClassName = css({
-  h: '80vh',
-  overflowY: 'auto !important',
+  h: '72vh',
```
Comment (rifont) [inline apps/web/src/studio/components/workflows/step-editor/WorkflowStepEditorControlsPanel.tsx:145]: With this height at 100% zoom, the step controls was being clipped at the bottom of the page. This patches that. The real fix is to use `flex-grow` in more places across the app, but that's a job for a different PR.

Comment (rifont) [reply apps/web/src/studio/components/workflows/step-editor/WorkflowStepEditorControlsPanel.tsx:145]: Fix applied at 100% zoom:
image

Comment (antonjoel82) [reply apps/web/src/studio/components/workflows/step-editor/WorkflowStepEditorControlsPanel.tsx:145]: Agreed -- we have some reused or foundational components that broke true flex layouts at the base level, so unfortunately they propagated all the way through. I fixed them in the side nav, but did not go back and address them yet in the content area

File: libs/novui/src/global.styles.ts (hunk 4 above)
Comment (antonjoel82) [reply libs/novui/src/global.styles.ts]: ✏ **suggestion:** This is also a good code comment!‏

Comment (antonjoel82) [reply libs/novui/src/global.styles.ts]: Also, we should make B80 / B30 into its own semantic color token for `scrollbar`, and then should be able to just do a single template literal here

**Review by antonjoel82 — APPROVED** [review]
Thanks for the changes!

File: libs/novui/src/global.styles.ts (hunk 4 above)
Comment (rifont) [reply libs/novui/src/global.styles.ts]: Great points, I've created new semantic tokens and updated to use those tokens throughout the global css file.

**Pull request merged**